installation/README.adoc

```diff
@@ -0,0 +1,204 @@
+Installation
+------------
+
+KubeVirt is a virtualization add-on to Kubernetes and this guide assumes
+that a Kubernetes cluster is already installed.
+
+Requirements
+~~~~~~~~~~~~
+
+A few requirements need to be met before you can begin:
+
+* https://kubernetes.io[Kubernetes] cluster
+(https://github.com/openshift/origin[OpenShift], Tectonic)
+* `kubectl` client utility
+* `git`
+
+Minimum Cluster Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Kubernetes 1.10 or later is required to run KubeVirt.
+
+In addition it can be that feature gates need to be opened.
+
+Runtime
++++++++
+
+KubeVirt is currently support on the following container runtimes:
+
+* docker
+* crio (with runv)
+
+Virtualization support
+^^^^^^^^^^^^^^^^^^^^^^
+
+There are several distributions of Kubernetes, you need to decide on one
+and deploy it.
+
+Hardware with virtualization support is recommended. You can use
+virt-host-validate to ensure that your hosts are capable of running
+virtualization workloads:
+
+[source,bash]
+----
+$ virt-host-validate qemu
+  QEMU: Checking for hardware virtualization                                 : PASS
+  QEMU: Checking if device /dev/kvm exists                                   : PASS
+  QEMU: Checking if device /dev/kvm is accessible                            : PASS
+  QEMU: Checking if device /dev/vhost-net exists                             : PASS
+  QEMU: Checking if device /dev/net/tun exists                               : PASS
+...
+----
+
+Software emulation
+++++++++++++++++++
+
+If hardware virtualization is not available, then a
+https://github.com/kubevirt/kubevirt/blob/master/docs/software-emulation.md[software
+emulation fallback] can be enabled using:
+
+....
+$ kubectl configmap -n kube-system kubevirt-config \
```

Comment:
Not sure how we handle this... but fyi: I just opened  a PR on the md file to fix this command to `kubectl create configmap...`